controllers/testclustertopology_controller.go

```diff
@@ -517,14 +530,15 @@ func (r *TestClusterTopologyReconciler) checkTiKVFinishedScaleIn(ctx context.Con
 	ct.Status.TiDBClusterInfo.OfflineList = offlineList
 	if len(pendingOfflineList) != 0 || len(offlineList) != 0 {
 		log.Info("cluster is trying prune", "PendingOfflineList", pendingOfflineList, "OfflineList", offlineList)
-		return false, ctrl.Result{RequeueAfter: time.Second * 10}, err
+		return false,  nil
 	}
 
 	err = tiupCtl.PruneCluster(log, ct.Name, ct)
 	if err != nil {
-		return false, ctrl.Result{}, err
+		return false, err
 	}
-	return true, ctrl.Result{}, nil
+	ct.Status.TiDBClusterInfo.IsScale = false
```

Comment:
IMO it's redundant because the method is called only if `ct.Status.TiDBClusterIn.isScale == false`